aws.yaml

```diff
@@ -1,3 +1,23 @@
+- active: false
+  authorities:
+    - endpoint: http://app-operator:8000
+      name: app-operator
+      version: 1.0.0
+    - endpoint: http://aws-operator:8000
+      name: aws-operator
+      version: 5.5.0
+    - endpoint: http://cert-operator:8000
+      name: cert-operator
+      version: 0.1.0
+    - endpoint: http://chart-operator:8000
+      name: chart-operator
+      version: 0.7.0
+    - endpoint: http://cluster-operator:8000
+      name: cluster-operator
+      provider: aws
+      version: 0.20.0
```

Comment:
Does 0.21.0 already exist?

Reply:
I don't see it wired here https://github.com/giantswarm/cluster-operator/blob/master/service/controller/aws/legacy_cluster.go 
@rossf7 correct me if I'm wrong

Reply:
I'm about to merge last PR to add 0.21.0. I just want to test it on ghost first.

https://github.com/giantswarm/cluster-operator/pull/685

Reply:
Done.

Reply:
updated, ptal